Using TensorFlow backend.
Traceback (most recent call last):
  File "Main.py", line 8, in <module>
    from TestClass import *
ModuleNotFoundError: No module named 'TestClass'
Using TensorFlow backend.
[94mObteniendo datos...[0m
[93m[AVISO] Cargando datos generados previamente...[0m
[94mCreando modelo...[0m
1	0,1	512	3,1657661393062915	67,32179141044617
2	0,1	512	3,1657661393062915	57,723917961120605
--------------------------------------------------
1	0,001	512	3,1657661393062915	59,83234667778015
2	0,001	512	3,1657661393062915	57,70427107810974
--------------------------------------------------
1	1e-05	512	0,940832167625839	60,00221657752991
2	1e-05	512	0,9729652011108892	57,25858473777771
3	1e-05	512	0,9510745472406791	57,38513898849487
4	1e-05	512	0,9913145472260806	57,79359674453735
5	1e-05	512	1,081370027307658	57,937127351760864
6	1e-05	512	1,216453577127463	58,05201292037964
7	1e-05	512	1,3700599557196067	58,15068531036377
--------------------------------------------------
1	1e-07	512	0,6920191362458379	59,7741539478302
2	1e-07	512	0,6921220049857064	58,09332871437073
3	1e-07	512	0,6922225790561056	57,990476846694946
4	1e-07	512	0,6923215103128829	57,90551042556763
5	1e-07	512	0,6924189969450203	58,121480226516724
6	1e-07	512	0,6925153252683756	58,016286849975586
7	1e-07	512	0,6926101257547354	57,68834924697876
--------------------------------------------------
1	1e-09	512	0,6935796919585355	60,365195989608765
2	1e-09	512	0,693580370535481	56,95709228515625
3	1e-09	512	0,6935810336484113	57,603785037994385
4	1e-09	512	0,6935817028465987	57,462119579315186
5	1e-09	512	0,6935823671945918	57,69021964073181
6	1e-09	512	0,6935830366243535	57,638267993927
7	1e-09	512	0,6935837005606589	57,50371050834656
--------------------------------------------------
